kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/aws/s3/sink/S3SinkTaskAvroEnvelopeTest.scala

```diff
@@ -84,7 +84,7 @@ class S3SinkTaskAvroEnvelopeTest
     record
   }
 
-  "S3SinkTask" should "write to avro format using V1 format" in {
+  /*"S3SinkTask" should "write to avro format" in {
```

Comment:
Is this test case no longer valid?  Can it be removed?  Does the fact it needs to be commented out represent a breaking change that will need to be communicated to customers?

Reply:
sorry, it is needed, I commented to test only the other test added. my mistake